packages/twenty-front/src/pages/settings/serverless-functions/SettingsServerlessFunctionDetail.tsx

```diff
@@ -72,7 +68,7 @@ export const SettingsServerlessFunctionDetail = () => {
     }
   };
 
-  const handleSave = useDebouncedCallback(save, 500);
+  const handleSave = usePreventOverlapCallback(save, 1000);
```

Comment:
so the goal of this function was preventing the double save?

Reply:
yes, save can take several seconds when using lambdas, and it returns an error if you update an updating function